netsuitesdk/api/vendor_bills.py

```diff
@@ -16,7 +18,6 @@ class VendorBills(ApiBase):
         'memo',
         'tranDate',
         'tranId',
-        'itemList',
```

Comment:
Looks like the changes here are to support customFields for items. We don't use item based lines, so we're in the clear. 😄